charts/posthog/values.yaml

```
@@ -1763,6 +1763,71 @@ prometheus:
       # https://github.com/samber/awesome-prometheus-alerts
       #
       groups:
+        - name: PostHog
+          rules:
+            - alert: ExportsConsumerLagIncreasing
+              expr: sum(delta(kafka_consumergroup_lag{topic="clickhouse_events_json", consumergroup="clickhouse-plugin-server-async"}[10m])) > 0
+              for: 10m
+              labels:
+                severity: critical
+              annotations:
+                summary: Exports and WebHooks backlog appears to have been increasing for 10 minutes straight.
+                description: "The backlog on consumer group clickhouse-plugin-server-async for topic clickhouse_events_json is increasing."
+
+            - alert: ClickHouseEventsConsumerLagIncreasing
+              expr: sum(delta(kafka_consumergroup_lag{topic="clickhouse_events_json", consumergroup="group1"}[10m])) > 0
+              for: 10m
+              labels:
+                severity: critical
+              annotations:
+                summary: ClickHouse events ingestion lag has been increasing for 10 minutes straight.
+                description: "The backlog on consumer group group1 for topic clickhouse_events_json is increasing."
+
+            - alert: ClickHousePersonConsumerLagIncreasing
+              expr: sum(delta(kafka_consumergroup_lag{topic="clickhouse_person", consumergroup="group1"}[10m])) > 0
+              for: 10m
+              labels:
+                severity: critical
+              annotations:
+                summary: ClickHouse person ingestion lag has been increasing for 10 minutes straight.
+                description: "The backlog on consumer group group1 for topic clickhouse_person is increasing."
+
+            - alert: ClickHousePersonConsumerLagIncreasing
```

```suggestion
            - alert: ClickHouseGroupsConsumerLagIncreasing
```

Copy-pasta error, due to too many identical alerts ^^ Maybe merge the 4 `group1` ones, with `sum(delta(kafka_consumergroup_lag{topic=~"clickhouse_events_json|clickhouse_person|clickhouse_groups|clickhouse_person_distinct_id", consumergroup="group1"}[10m])) by (topic)`, if we already have monitors that alert on different contexts

---

Yeah I was keeping separate just in case it was useful to set more descriptive summaries/descriptions, but I think it doesn't.